pkgs/servers/monitoring/nagios/plugins/smartmon.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python2
+, smartmontools
+}:
+
+stdenv.mkDerivation {
+  pname = "check_smartmon";
+  version = "master";
```

When targeting something without a clear release version, you use something like

```suggestion
  version = unstable-YYYY-MM-DD;
```
where YYYY-MM-DD is the date the code was released. 

In this particular case, it would be:

```suggestion
  version = unstable-2017-02-05;
```